MapboxNavigation/InstructionsBannerViewLayout.swift

```diff
@@ -0,0 +1,121 @@
+import UIKit
+
+extension InstructionsBannerView {
+    
+    func commonInit() {
+        setupViews()
+        setupLayout()
+        setupAvailableBounds()
+    }
+    
+    func setupViews() {
+        backgroundColor = .clear
+        
+        let maneuverView = ManeuverView()
+        maneuverView.backgroundColor = .clear
+        maneuverView.translatesAutoresizingMaskIntoConstraints = false
+        addSubview(maneuverView)
+        self.maneuverView = maneuverView
+        
+        let distanceLabel = DistanceLabel()
+        distanceLabel.translatesAutoresizingMaskIntoConstraints = false
+        distanceLabel.adjustsFontSizeToFitWidth = true
+        distanceLabel.minimumScaleFactor = 0.2
```

Comment:
The distance label probably shouldn't resize smaller than the default size `22`. If we were to have a min scale, it should be `20` at the smallest.